lte/gateway/Vagrantfile

```diff
@@ -37,7 +37,8 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
     # using a specific IP.
     magma.vm.network "private_network", ip: "192.168.60.142", nic_type: "82540EM"
     # iperf3 trfserver routable IP.
-    magma.vm.network "private_network", ip: "192.168.129.1", nic_type: "82540EM"
+    magma.vm.network "private_network", ip: "3001::10", nic_type: "82540EM"
+    #magma.vm.network "private_network", ip: "192.168.129.1", nic_type: "82540EM"
```

Comment:
Why are we commenting IPv4 address here. Are we expecting data path to be only in IPv6 and not in both IPv4 and IPv6?